helm/korifi/kpack-image-builder/cluster-builder.yaml

```diff
@@ -17,7 +17,7 @@ kind: ClusterStack
 metadata:
   name: cf-default-stack
 spec:
-  id: "io.buildpacks.stacks.bionic"
+  id: {{ .Values.kpackImageBuilder.clusterStackID | quote }}
```

Comment:
Do users really need to configure this ID? I.e. is that not just a string? If so, we could use something like `cf.default.stack` or something. Reading kpack docs, I am not sure how `ClusterStack`'s `metadata.name` differs from `spec.id` and why do you need it at all

Reply:
The ["Create a Stack" docs](https://buildpacks.io/docs/operator-guide/create-a-stack/) mention it a bit:

> NOTE: The stack identifier implies compatibility with other stacks of that same identifier. For instance, a custom stack may use io.buildpacks.stacks.jammy as its identifier so long as it will work with buildpacks that declare compatibility with the io.buildpacks.stacks.jammy stack.

Reply:
Thanks, that makes sense now